data-raw/auxiliary_data_sets.R

```diff
@@ -30,8 +31,35 @@ unit_conversion <- dplyr::tribble(
   "power",       "MWh",             8.598e-08
 )
 
+
+# activity units
+
+library(dplyr)
```

Comment:
I wonder if this `library()` call was in there for the pipes.. maybe a good reason to remove the pipes and get rid of this call? seems a bit awkward

Reply:
yup, there just for the pipes... and I left it there rather than make more dramatic changes to the existing code I was copying in

Reply:
okay, maybe a follow up issue